docs/designs/loc.md

```diff
@@ -29,6 +29,18 @@ DocFX build supports localization contents, there are a few features need to be
   - Overwrite the configurations you want or use source configuration by default  
   - Focus the Loc publishing configuration controlling, like bilingual or contribution.  
   - Source configuration changing will be immediately applied to Loc, no more manually sync.
+  ```yaml
+  "[de-de,zh-cn,ja-jp,ko-kr]":
```

Comment:
it's also used for branch overwrite config like:
```
"[master, live]":{}
"[develop]": {}
```

Reply:
we need a way to tell if 'master' is a branch or a locale

Reply:
they are in different files, but using the same syntax